Makefile

```diff
@@ -515,6 +516,9 @@ ifneq ($(wildcard $(VERIFIC_DIR)/hier_tree),)
 VERIFIC_COMPONENTS += hier_tree
 endif
 endif
+ifeq ($(VERIFIC_LINEFILE_INCLUDES_LOOPS),1)
+CXXFLAGS += -DVERIFIC_LINEFILE_INCLUDES_LOOPS
```

Comment:
I don't understand how this define gets propagated to Verific, which has a separate build process.

Reply:
Fixed by Verific define